plugins/logs/status/status.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2023 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+package status
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/open-policy-agent/opa/metrics"
+)
+
+const (
+	errCode = "decision_log_error"
+)
+
+// Status represents the status of processing a decision log.
+type Status struct {
+	Code     string          `json:"code,omitempty"`
+	Message  string          `json:"message,omitempty"`
+	HTTPCode json.Number     `json:"http_code,omitempty"`
```

Comment:
A plain int would do, too, wouldn't it?

Reply:
It should but just wanted to use what we have [here](https://sourcegraph.com/github.com/open-policy-agent/opa@8aa60e8b525f060ded2c31bba15f11bf82f7eb7a/-/blob/plugins/bundle/status.go?L37).

Reply:
Ok!